server/views/pages/overview/partials/overviewTabActionPlanColumn.njk

```diff
@@ -30,7 +30,10 @@
   {% for goal in actionPlan.goals %}
     <div class="govuk-summary-card" data-qa="goal-summary-card">
       <div class="govuk-summary-card__title-wrapper">
-        <h3 class="govuk-summary-card__title">Goal {{ loop.index }}</h3>
+        <h3 class="govuk-summary-card__title">Goal {{ loop.index }}
+          <span class="govuk-caption-m">Created on {{ goal.createdAt | formatDate('DD MMMM YYYY') }} <span class="govuk-!-display-none-print">by {{ goal.createdByDisplayName }}</span>,
```

Comment:
Minor, but could you change the date format to `D MMMM YYYY` please? (for both of them)
By doing that it will drop the leading `0` in days 1-9 (but for days >= 10 then it correctly renders as 2 digits 👍 )

Reply:
Updated